src/Profile/Request/ExtraData/ThirdParty/SandboxIssuingAttributes.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yoti\Sandbox\Profile\Request\ExtraData\ThirdParty;
+
+use Yoti\Util\DateTime;
+use Yoti\Util\Validation;
+
+class SandboxIssuingAttributes implements \JsonSerializable
+{
+    /**
+     * @var \DateTime
+     */
+    private $expiryDate;
+
+    /**
+     * @var SandboxDefinition[]
+     */
+    private $definitions;
+
+    /**
+     * @param \DateTime $expiryDate
+     * @param SandboxDefinition[] $definitions
+     */
+    public function __construct(\DateTime $expiryDate, array $definitions)
+    {
+        $this->expiryDate = $expiryDate;
+
+        Validation::isArrayOfType($definitions, [SandboxDefinition::class], 'definitions');
+        $this->definitions = $definitions;
+    }
+
+    /**
+     * @return \stdClass
+     */
+    public function jsonSerialize(): \stdClass
+    {
+        return (object) [
+            'expiry_date' => $this->expiryDate->format(DateTime::RFC3339),
```

Comment:
We should change this to `DATE_RFC3339_EXTENDED` inline with format expected by the dynamic QR service

Reply:
Fixed in 886d36601036db88f7d015822345cbed1a99947a